### PR TITLE
Add Window Segments Enumeration API

### PR DESCRIPTION
**Pull request by anssiko**
[HTML preview](https://raw.githack.com/webscreens/cg-charter/window-segments/index.html)
[HTML diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwebscreens.github.io%2Fcg-charter%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fwebscreens%2Fcg-charter%2Fwindow-segments%2Findex.html)

### CODE REVIEW

**Comment by anssiko** [comment]
This has been [announced](https://lists.w3.org/Archives/Public/public-webscreens/2020Jan/0000.html) on the Second Screen CG mailing list as follows:

>Hi Second Screen CG participants,
>Cc Second Screen WG, Zouhir, Daniel,
>
>This is a Call for Consensus (CfC) to adopt the Window Segments Enumeration API [1] incubation into the Second Screen Community Group.
>
>This API proposal is championed by Microsoft's Zouhir and Daniel (cc'd) who are in their own words very keen to work with this community.
>
>Please review the proposed revised charter [2] and signal your support by adding a comment or a thumbs up (+1) reaction to the Pull Request [3], or alternatively, by responding to this message by 7 February 2020. All feedback is welcome.
>
>Thanks,
>
>-Anssi (Second Screen WG/CG chair)
>
>[1] https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Foldables/explainer.md#proposal-window-segments-enumeration-javascript-api
>
>[2] https://raw.githack.com/webscreens/cg-charter/window-segments/index.html
>
>[3] https://github.com/webscreens/cg-charter/pull/21

**Comment by anssiko** [comment]
You can add your 👍 here to signal support.

File: index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h1>
         Last Modified:
       </dt>
       <dd>
-        1 January 2020
+        7 February 2020
       </dd>
     </dl>
     <h2 id="goals">
```
Comment (michaelwasserman) [inline index.html:63]: The goals section should include Window Segments in the incubations list with Screen Enumeration and Window Placement.

Comment (anssiko) [reply index.html:63]: Good catch @michaelwasserman! I pushed a fix.

**Review by markafoltz — APPROVED** [review]

**Comment by travisleithead** [comment]
Assuming this gets the required support, I'd like to suggest we create a new repo under [webscreens](https://github.com/webscreens/) and copy the relevant parts of the explainer here for future development. Then perhaps the references in the charter text can refer to that URL rather than the one at the [MicrosoftEdge](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Foldables/explainer.md) repo.

Any suggestions on a repo name? @zouhir @dlibby-

**Comment by anssiko** [comment]
This CfC passed with no concerns and explicit signals for support.

I'll work with Microsoft reps to migrate the relevant parts of the explainer into a new repo under the webscreens GH org that hosts also other incubations, and update the links in the charter as suggested by @travisleithead.

**Pull request merged**